presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/environment/MultinodeTlsKerberos.java

```diff
@@ -0,0 +1,101 @@
+import io.prestosql.tests.product.launcher.env.DockerContainer;
+import io.prestosql.tests.product.launcher.env.Environment;
+import io.prestosql.tests.product.launcher.env.EnvironmentOptions;
+import io.prestosql.tests.product.launcher.env.common.AbstractEnvironmentProvider;
+import io.prestosql.tests.product.launcher.env.common.Hadoop;
+import io.prestosql.tests.product.launcher.env.common.Kerberos;
+import io.prestosql.tests.product.launcher.env.common.Standard;
+import org.testcontainers.containers.startupcheck.IsRunningStartupCheckStrategy;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import javax.inject.Inject;
+
+import java.io.File;
+import java.time.Duration;
+import java.util.Objects;
+
+import static com.google.common.base.Verify.verify;
+import static io.prestosql.tests.product.launcher.env.common.Hadoop.CONTAINER_PRESTO_HIVE_PROPERTIES;
+import static io.prestosql.tests.product.launcher.env.common.Standard.CONTAINER_PRESTO_CONFIG_PROPERTIES;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.containers.BindMode.READ_ONLY;
+
+public class MultinodeTlsKerberos
+        extends AbstractEnvironmentProvider
+{
+    private final PathResolver pathResolver;
+    private final DockerFiles dockerFiles;
+
+    private final String prestoDockerImageName;
+    private final File serverPackage;
+
+    @Inject
+    public MultinodeTlsKerberos(
+            PathResolver pathResolver,
+            DockerFiles dockerFiles,
+            Standard standard,
+            Hadoop hadoop,
+            Kerberos kerberos,
+            EnvironmentOptions environmentOptions)
+    {
+        super(ImmutableList.of(standard, hadoop, kerberos));
+        this.pathResolver = requireNonNull(pathResolver, "pathResolver is null");
+        this.dockerFiles = requireNonNull(dockerFiles, "dockerFiles is null");
+        String hadoopBaseImage = requireNonNull(environmentOptions.hadoopBaseImage, "environmentOptions.hadoopBaseImage is null");
+        String imagesVersion = requireNonNull(environmentOptions.imagesVersion, "environmentOptions.imagesVersion is null");
+        prestoDockerImageName = hadoopBaseImage + "-kerberized:" + imagesVersion;
+        serverPackage = requireNonNull(environmentOptions.serverPackage, "environmentOptions.serverPackage is null");
+    }
+
+    @Override
+    @SuppressWarnings("resource")
+    protected void extendEnvironment(Environment.Builder builder)
+    {
+        super.extendEnvironment(builder);
+
+        builder.configureContainer("presto-master", container -> {
+            verify(Objects.equals(container.getDockerImageName(), prestoDockerImageName), "Expected image '%s', but is '%s'", prestoDockerImageName, container.getDockerImageName());
+            container
+                    .withFileSystemBind(dockerFiles.getDockerFilesHostPath("conf/environment/multinode-tls-kerberos/config-master.properties"), CONTAINER_PRESTO_CONFIG_PROPERTIES, READ_ONLY)
```

Comment:
It'd be helpful to extract a constant for the config prefix (perhaps clean this up separately in all of the launchers)

Reply:
i am not totally sure. a constant would make paths less searchable

also, we can easily use constants for loading resources from host into docker
but we can't use constants withing dockerized scripts; so this makes the situation assymetric
i am not confident which direction is the best yet

Reply:
I was thinking to remove the duplication of `conf/environment/multinode-tls-kerberos` within this file.

Reply:
that's how i undestand this. but then, if someone wants to find where is `config-master.properties` referenced... it's ambiguous (several files of this name). and if they search w/ path suffix, they will find none.

that's just a downside. upside is less repetition, i admit

i choose to keep it for now.